noxfile.py

```diff
@@ -7,8 +7,12 @@
 import nox
 
 # Options to modify nox behaviour
+nox.options.default_venv_backend = "uv|virtualenv"
```

Comment:
Fixed my issues locally using `uv` https://github.com/astral-sh/uv/issues/6579

Reply:
I remember we could not use `uv` because of https://github.com/astral-sh/uv/issues/1794

I am surprised that it is working now. We still use the `--prefer-binary` flag in our tests -

https://github.com/glass-dev/glass/blob/f6cfde4487c0d1c1c155f765d2cc6da48fa9a911/.github/test-constraints.txt#L10